composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=5.4.0",
         "league/flysystem": "~1.0",
-        "aws/aws-sdk-php": "3.*@dev"
+        "aws/aws-sdk-php": "^3.0.0"
```

Comment:
Ok.

I think we'll soon see `^` be used much more often. Ofc gradually, it makes no sense to change all existing version constraints.